ArduPlane/Attitude.cpp

```diff
@@ -456,15 +444,18 @@ void Plane::stabilize()
         stabilize_training(speed_scaler);
     } else if (control_mode == &mode_acro) {
         stabilize_acro(speed_scaler);
-    } else if ((control_mode == &mode_qstabilize ||
-                control_mode == &mode_qhover ||
-                control_mode == &mode_qloiter ||
-                control_mode == &mode_qland ||
-                control_mode == &mode_qrtl ||
-                control_mode == &mode_qacro ||
-                control_mode == &mode_qautotune) &&
-               !quadplane.in_tailsitter_vtol_transition(now)) {
-        quadplane.control_run();
+    } else if (control_mode->is_vtol_mode() && !quadplane.in_tailsitter_vtol_transition(now)) {
+        if (quadplane.initialised) {
```

Comment:
I'd vote for removing the "if (quadplane.initialised) {" part because this means you could potentially not run control_mode->run().

I'm not the expert on quadplane though obviously.